src/applications/pensions/tests/containers/UnauthenticatedWarningAlert.unit.spec.jsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import sinon from 'sinon';
+import { render, fireEvent, waitFor } from '@testing-library/react';
+import { expect } from 'chai';
+import UnauthenticatedWarningAlert from '../../containers/UnauthenticatedWarningAlert';
+
+const isNotLoggedInStore = ({ isLoggedIn = false } = {}) => ({
+  getState: () => ({
+    user: {
+      login: {
+        currentlyLoggedIn: isLoggedIn,
+      },
+    },
+  }),
+  subscribe: () => {},
+  dispatch: () => {},
+});
+
+const isLoggedInStore = ({ isLoggedIn = true } = {}) => ({
```

Comment:
You could rename this `currentlyLoggedIn` and just pass the parameter = true/false when you call it that way you don't have to write it twice

Reply:
Good call.